daemon/daemon.go

```diff
@@ -1147,7 +1147,7 @@ func NewDaemon(dp datapath.Datapath) (*Daemon, *endpointRestoreState, error) {
 		}
 
 		opts := make(map[workloads.WorkloadRuntimeType]map[string]string)
-		for _, rt := range option.Config.Workloads {
+		for rt := range option.Config.ContainerRuntimeEndpoint {
```

Comment:
Thanks for fixing. Maybe we can merge both for loops into one?

Reply:
yep, makes sense